content/includes/header.html

```diff
@@ -20,11 +20,7 @@
 <meta content="https://www.vets.gov" property="og:title">
 {% endif %}
 
-{% if title %}
```

Comment:
Ok - whatever you think is right! I was reading the issue as "remove the conditional because it should always be x in all cases".